### ISSUE TITLE
We need correct checkpoints

### ISSUE THREAD

**Issue by Oktai15** [akamaster/pytorch_resnet_cifar10#4]
Hi, @akamaster! Great work!

Actually, for comparing of results we need to train ResNet's on 45k (train), choose best on 5k (validation) and write as result accuracy on 10k (test). Your better results mean that you slightly "overfitting" on test, because you chose best model by test and it is not correctly at all. So, if it is possible then re-train models on 45k/5k.

**Comment by akamaster** [comment]
Thanks for suggestion, I know about it and I don't have time to re-train or do something new with this repository yet. You are welcome to retrain the models using validation split and make a pull request, I would happily update current models or add your models to accompany them. Hope that helps, Yerlan
